item-spec/item-spec.md

```diff
@@ -299,6 +299,13 @@ As detailed above, Items contain properties, which are the main source of metada
 extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for
 a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no
 single value for all assets.
+If a property is defined in the Item Properties, it is the default value for all assets in the Item.
+If a property is defined in a Collection on the top-level, it is the default value for all assets in the Collection.
```

Comment:
> it is the default value for all assets in the Collection

Not items? E.g. if a `Collection` has `proj:epsg=32613`, does that projection information apply to the item (e.g. in the case of a `proj:geometry` on the item).

Reply:
No, we only inherit in the same file.